src/everest/detached/jobs/everserver.py

```diff
@@ -159,6 +162,7 @@ def get_opt_progress(
         ssl_certfile=server_config["cert_path"],
         ssl_version=ssl.PROTOCOL_SSLv23,
         ssl_keyfile_password=server_config["key_passwd"],
+        log_level=logging.CRITICAL,
```

Comment:
:heart: 

The messages from the HTTP server were driving me crazy during debugging